sdk/eventhub/eventhubs-checkpointstore-blob/samples/receiveEventsUsingCheckpointStore.ts

```diff
@@ -0,0 +1,77 @@
+/*
```

Comment:
Since this sample is already under the checkpoint store library, isnt it redundant to name the file to say that it uses the checkpoint store?

Reply:
It's true but...this samples comes from the event-hubs library. So I'd like to keep the name the same to make that at least semi-obvious until we come up with a better syncing solution. :)